pymrcpp/pyMethods.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ *  \date Mar 20, 2018
+ *  \author Magnar Bjørgve <magnar.bjorgve@uit.no> \n
+ *          Hylleraas Centre for Quantum Molecular Sciences \n
+ *          UiT - The Arctic University of Norway
+ */
+
+#include "pybind11/pybind11.h"
+#include "pybind11/functional.h"
+
+#include "trees/FunctionTree.h"
+#include "trees/FunctionTreeVector.h"
+#include "operators/DerivativeOperator.h"
+#include "operators/ConvolutionOperator.h"
+#include "treebuilders/add.h"
+#include "treebuilders/multiply.h"
+#include "treebuilders/apply.h"
+#include "treebuilders/grid.h"
+#include "treebuilders/project.h"
+#include "pyProject.h"
+
+namespace py = pybind11;
+using namespace mrcpp;
+
+void pyProject1D(py::module &m) {
+    m.def("project", py::overload_cast<double, FunctionTree<1> &, std::function<double (double)>, int>(&project1D),
```

Comment:
Also, is `std::function<ReturnType (ArgumentTypes...)>` always an alias for `RepresentableFunction<D>`? Then I would use a template alias: `using Function3D = RepresentableFunction<3>` rather than `std::function`.